usaspending_api/awards/v2/data_layer/orm_mappers.py

```diff
@@ -50,7 +51,7 @@
         ("description", "description"),
         ("total_obligation", "total_obligation"),
         ("base_exercised_options_val", "base_exercised_options_val"),
```

Comment:
I overlooked adding `base_exercised_options_val ` -> `base_exercised_options` to the API work